mingw-w64-git/PKGBUILD

```diff
@@ -74,20 +74,25 @@ sha256sums=('SKIP'
 
 USE_ASCIIDOCTOR="YesPlease"
 COMPAT_CFLAGS=
+STRIP=
+STRIP_OPTS=
 
 if [[ "$MSYSTEM" == CLANG* ]]
 then
   if test -z "$WITHOUT_PDBS"
   then
     pkgname+=("${MINGW_PACKAGE_PREFIX}-${_realname}-pdb")
     options+=('!strip')
+    STRIP=llvm-strip
```

Comment:
One sentence in the commit message is a bit mixed up/misleading; LLVM based toolchains do have a tool named llvm-strip, but it's not the llvm-strip tool that generates the pdb files, it's the lld linker.

Reply:
Good catch, thanks! Have updated the commit messages.